src/scripts/helpers/backbone/views/base.coffee

```diff
@@ -77,11 +77,12 @@ define (require) ->
         historyPage = linksHelper.getCurrentPathComponents().page
 
       if @pageTitle
-        document.title = @pageTitle + settings.titleSuffix
+        pageTitle = @pageTitle.replace(/<\/?[^>]+(>|$)/g, '') # Strip tags for baked HTML titles
+        document.title = pageTitle + settings.titleSuffix
```

Comment:
The only downside I can think of is that this might cause untrusted code (`<script>alert("Hacked!")</script>`) to be executed in the browser if that is in the title (not 100% sure on this)